iOS Example Frame/iOS Example Frame/Style/MatrixStyle.swift

```diff
@@ -199,6 +185,29 @@ private enum StyleOrganiser {
         var error: ElementErrorViewStyle? = ErrorViewStyle(text: "Insert a valid security code")
     }
 
+    struct PayButtonSection: ElementButtonStyle {
```

Comment:
This is actually a good change, but I don't think "Section makes sense when looking at the button.

User inputs are sections because they can be composed of:
- title
- subtitle
- optional mark
- input
- error

This is not the case for the button, which is just a button.

```suggestion
    struct PayButtonStyle: ElementButtonStyle {
```